### DISCUSSION TITLE
✨ Jitsi Meet

### DISCUSSION THREAD

**Discussion by JigSawFr** [runtipi/runtipi-appstore#2196] [Requests]
### Description

Jitsi Meet is a set of Open Source projects which empower users to use and deploy video conferencing platforms with state-of-the-art video quality and features.

Amongst others here are the main features Jitsi Meet offers:

- Support for all current browsers
- Mobile applications
- Web and native SDKs for integration
- HD audio and video
- Content sharing
- Raise hand and reactions
- Chat with private conversations
- Polls
- Virtual backgrounds

And many more!

### Official Website

https://jitsi.org/meet

### Documentation link

https://jitsi.github.io/handbook/docs/devops-guide/devops-guide-docker

### Last application release & date

2.0.9164 (20/12/2023)

### Application license

Apache-2.0 license

### Source code repository link

https://github.com/jitsi/jitsi-meet

### Docker image link

https://hub.docker.com/u/jitsi/

### Please confirm the following

- [X] I confirm that I've put the app name in the title of the discussion
- [X] I confirm that the application is actively maintained.
- [X] I confirm that the docker image of the application is actively maintained.
- [X] I confirm that I have indeed checked that no similar request for this application already exists

### Gathering crowds

Hey folks!

Please upvote 👍 this discussion to show your interest in this request! 

Thanks ⛺

**Comment by JigSawFr** [comment]
Originally requested by @RasheedAZ. Upvoted by @Nurullaev.

Waiting on hooks.

**Comment by jasonpearce** [comment]
If helpful, here is Jitsi Meet packaged for YunoHost:

https://github.com/YunoHost-Apps/jitsi_ynh
